src/pages/index.js

```diff
@@ -71,22 +104,35 @@ imageModal.setEventListeners();
 
 // #region Cards rendering
 
-const cards = document.querySelector(".cards");
+// Is there a better practice for defining a global variable inside a promise resolve?
+let cardsSection = null;
+
+api.getCards()
+    .then((res) => {
+        const cards = Array.from(res).reverse();
+        
+        cardsSection = new Section({
+            items: cards,
+            renderer: (data) => {
+                const card = getCardElement(data);
+                cardsSection.addItem(card);
+            }
+        }, ".cards");
+
+        cardsSection.renderItems();
+    });
 
-const cardsSection = new Section({
-    items: initialCards,
-    renderer: (data) => {
-        const card = getCardElement(data);
-        cardsSection.addItem(card);
-    }
-}, ".cards");
+const deletePopup = new Popup("#confirm-modal");
+deletePopup.setEventListeners();
 
-cardsSection.renderItems();
+let deleteHandler = () => {};
+const deleteButton = document.querySelector("#delete-button");
+deleteButton.addEventListener("click", deleteHandler);
```

Comment:
This listener should be added inside `PopupWithConfirmation` according to the check list